suite-common/wallet-core/src/firmware/firmwareActions.ts

```diff
@@ -17,9 +17,19 @@ const setHashInvalid = createAction(
     }),
 );
 
-const setError = createAction(`${FIRMWARE_MODULE_PREFIX}/set-error`, (payload?: string) => ({
```

Comment:
I renamed `setError` to a more descriptive `setFirmwareUpdateError`, I verified it is not used for anything else that FW hash check on FW update